connector/src/main/java/org/jboss/as/connector/services/resourceadapters/IronJacamarActivationResourceService.java

```diff
@@ -0,0 +1,302 @@
+import org.jboss.as.connector.subsystems.resourceadapters.IronJacamarResourceCreator;
+import org.jboss.as.connector.subsystems.resourceadapters.IronJacamarResourceDefinition;
+import org.jboss.as.connector.subsystems.resourceadapters.ResourceAdaptersExtension;
+import org.jboss.as.controller.AttributeDefinition;
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.controller.PathElement;
+import org.jboss.as.controller.ResourceBuilder;
+import org.jboss.as.controller.descriptions.ModelDescriptionConstants;
+import org.jboss.as.controller.descriptions.OverrideDescriptionProvider;
+import org.jboss.as.controller.descriptions.StandardResourceDescriptionResolver;
+import org.jboss.as.controller.registry.ManagementResourceRegistration;
+import org.jboss.as.controller.registry.Resource;
+import org.jboss.dmr.ModelNode;
+import org.jboss.jca.core.api.management.AdminObject;
+import org.jboss.jca.core.api.management.ConnectionFactory;
+import org.jboss.jca.core.connectionmanager.ConnectionManager;
+import org.jboss.jca.core.spi.statistics.StatisticsPlugin;
+import org.jboss.jca.deployers.common.CommonDeployment;
+import org.jboss.msc.inject.Injector;
+import org.jboss.msc.service.Service;
+import org.jboss.msc.service.StartContext;
+import org.jboss.msc.service.StartException;
+import org.jboss.msc.service.StopContext;
+import org.jboss.msc.value.InjectedValue;
+
+public class IronJacamarActivationResourceService implements Service<ManagementResourceRegistration> {
+
+    private static PathElement SUBSYSTEM_PATH_ELEMENT = PathElement.pathElement(ModelDescriptionConstants.SUBSYSTEM, ResourceAdaptersExtension.SUBSYSTEM_NAME);
+    private static PathElement IJ_PATH_ELEMENT = PathElement.pathElement(Constants.IRONJACAMAR_NAME, Constants.IRONJACAMAR_NAME);
+
+
+    private final ManagementResourceRegistration registration;
+    private final Resource deploymentResource;
+    private final boolean statsEnabled;
+
+    protected final InjectedValue<ResourceAdapterDeployment> deployment = new InjectedValue<>();
+
+    protected final InjectedValue<AS7MetadataRepository> mdr = new InjectedValue<AS7MetadataRepository>();
+
+    /**
+     * create an instance *
+     */
+    public IronJacamarActivationResourceService(final ManagementResourceRegistration registration, final Resource deploymentResource,
+                                                final boolean statsEnabled) {
+        super();
+        this.registration = registration;
+        this.deploymentResource = deploymentResource;
+        this.statsEnabled = statsEnabled;
+    }
+
+
+    @Override
+    public void start(StartContext context) throws StartException {
+        ManagementResourceRegistration subRegistration;
+        final CommonDeployment deploymentMD = deployment.getValue().getDeployment();
+        ROOT_LOGGER.infof("Starting IronJacamarActivationResourceService %s", deploymentMD.getDeploymentName());
```

Comment:
this should be debugf otherwise it needs to be localized, but i think it is more debug message that anything else.